templates/snmp_exporter.service.j2

```diff
@@ -17,5 +17,24 @@ KillMode=process
 SyslogIdentifier=snmp_exporter
 Restart=always
 
+CapabilityBoundingSet=CAP_SET_UID
+LockPersonality=true
+NoNewPrivileges=true
+MemoryDenyWriteExecute=true
+PrivateTmp=true
+ProtectHome=true
+RemoveIPC=true
+RestrictSUIDSGID=true
+
+{% if snmp_exporter_systemd_version | int >= 232 %}
```

Comment:
I guess we should support this until [2020-06-30](https://wiki.debian.org/LTS).

Reply:
yeah...